clm/lm_wrapper.py

```diff
@@ -13,7 +13,7 @@
 from qanta.util.environment import QB_QUESTION_DB, QB_WIKI_LOCATION
 from qanta.util.constants import CLM_PATH, QB_SOURCE_LOCATION
 
-from clm import clm
+import clm
```

Comment:
I did. Was still getting the import error:
```
$ make clm
gcc -O3 `python3-config --includes` -std=c++11 -fPIC -c clm/clm.cpp -o clm/clm.o
swig -c++ -python clm/clm.swig
gcc -O3 `python3-config --includes` -std=c++11 -fPIC -c clm/clm_wrap.cxx -o clm/clm_wrap.o
g++ -shared `python3-config --ldflags` clm/clm.o clm/clm_wrap.o -o clm/_clm.so

$python3 clm/lm_wrapper.py
Traceback (most recent call last):
  File "clm/lm_wrapper.py", line 16, in <module>
    from clm import clm
ImportError: cannot import name 'clm'
```
Maybe it's because both lm_wrapper.py and clm.py are under clm/ ?